docs/layers/arc-layer.md

```diff
@@ -8,14 +8,16 @@
 
 The Arc Layer renders raised arcs joining pairs of source and target points,
 specified as latitude/longitude coordinates.
+[Source](https://github.com/uber/deck.gl/tree/master/src/layers/core/arc-layer)
```

Comment:
One of the problems we have had was that there was a lot of vertical empty space in the beginning of the layer docs. Every line with a tag, link etc pushes down the text about the props that 95% of the people are looking for. 

I'm open to discussion, but if we want it at the top, the layout (in the demo docs, not on github) is critical. The examples have a nice little link icon to the source if we could do that on the same line as the tags then it would be OK.